Libraries/CMSIS/5.9.0/Core/Include/core_rv32.h

```diff
@@ -318,14 +318,19 @@ typedef struct {
 
 /*@} end of group CMSIS_NVIC */
 
-#define MXC_BASE_INTR ((uint32_t)0xE5070000UL)
-//#define MXC_BASE_INTR                   ((uint32_t)0xE5000000UL)
+/* For MCUs with extra RISCV core (RISCV1) */
+#ifdef __riscv1
```

Comment:
Where does `__riscv1` get defined?

Reply:
This isn't relevant for the ME55 but will be for the ME13B, so there are no `__riscv1` references yet. This is preparation for ME13B since the ME55 is a subset of ME13B. Should I hold off on this change?

Reply:
Ok, yeah we might want to hold off until the system for the dual RISC-V cores is a little more fleshed out.  We should be able to check against a generic RISC-V compilation as well as a specific core.  So we may want to rename this to something like `__core_riscv1` and define the base one as `__core_riscv0`.  Right now our `RISCV_CORE` build option is just a toggle switch, but I can see it becoming a selector for the ME13B.

Reply:
Up to you.  I'm fine either way since it's unused